src/commands/mod.rs

```diff
@@ -24,6 +24,9 @@ pub mod set_master_password;
 pub mod rename;
 pub mod transfer;
 pub mod change;
+
```

Comment:
Please remove this space

Reply:
Sure thing. It's there for readability sale though

Reply:
@HackU6-Goats Oh, alright, well then feel free to let it there. Or best, if you can, run `cargo fmt` before committing, so we have standard styling everywhere :)